get-new-tool-versions/parsers/verify-added-to-image/verify-common-tool-parser.psm1

```diff
@@ -0,0 +1,25 @@
+function Search-ToolsVersionsNotOnImage {
+    param (
+        [string]$ToolName,
+        [string]$ReleasesUrl,
+        [string]$FilterParameter,
+        [string]$FilterArch
+    )
+    
+    $stableReleases = (Invoke-RestMethod $ReleasesUrl) | Where-Object stable -eq $true
```

Comment:
Why do we need ()?

Reply:
For some reason where-object sorting not work with invoke-restmethod without parentheses:
```
PS D:\Repo\Temp\versions-package-tools> (Invoke-RestMethod $ReleasesUrl | Where-Object stable -eq $true).count
0
PS D:\Repo\Temp\versions-package-tools> ((Invoke-RestMethod $ReleasesUrl) | Where-Object stable -eq $true).count
183
```